src/Nethermind/Nethermind.HonestValidator.Test/ValidatorClientTest.cs

```diff
@@ -55,6 +55,7 @@ public TestContext TestContext
         [Ignore("Disable test until OAPI is working again after 0.10.1 spec updates")]
         public async Task BasicSignBlockFromBeaconNodeProxy()
         {
+            await Task.Delay(0);
```

Comment:
I've stumbled across this [Task.Delay(0) not asynchronous](https://stackoverflow.com/questions/33407007/task-delay0-not-asynchronous)
dunno if this is related

Reply:
this is an ignored test - I think it is important not to lose focus on what is helpful in code review

Reply:
true, missed the Ignore atr

Reply:
Oh, but it is a test I should fix now that OAPI is working again.

Reply:
Card added to project board.